src/isadetection.c

```diff
@@ -48,6 +48,16 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <stdint.h>
 #include <stdlib.h>
 
+// Binaries produced by Visual Studio with solely AVX2 routines
+// can compile to AVX-512 thus causing crashes on non-AVX-512 systems.
+// This appears to affect VS 17.8 and 17.9. We disable AVX-512 and AVX2
+// on these systems. It seems that ClangCL is not affected.
```

Comment:
Might be worth a link to this PR for some more context, including the image miscompilation proof.

Reply:
Link to PR added in a commit.